frontstage/views/secure_messaging.py

```diff
@@ -306,14 +306,16 @@ def messages_get(session, label="INBOX"):
         raise ExternalServiceError(resp)
 
     response_data = json.loads(resp.text)
-    total_msgs = 0
 
-    for x in range(0, len(response_data['messages'])):
-        if "UNREAD" in response_data['messages'][x]["labels"]:
-            total_msgs += 1
+    labels = app.config['LABELS_GET_API_URL']
+
+    totals = requests.get(labels, headers=headers)
```

Comment:
We should have error handling on this request as we do for our other responses.
Check the response status code to check that it's right and handle whatever error status codes can be returned

Reply:
We should also add a test which covers what happens when we dont get the response we want from this request